catalog-info.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'platform-roadmap'
+  description: "This repository exists in order to hold tickets for the Open edX Platform Roadmap"
+  links:
+    - url: "http://github.com/openedx/platform-roadmap"
+      title: "Issue Tracker"
+      icon: "Issue"
```

Comment:
@feanil  how should we set this metadata for repos like this that don't hold source code?